main-command/src/main/contraband-scala/CommandSourceFormats.scala

```diff
@@ -1,5 +1,5 @@
 /**
- * This code is generated using [[http://www.scala-sbt.org/contraband/ sbt-contraband]].
+ * This code is generated using [[https://www.scala-sbt.org/contraband/ sbt-contraband]].
```

Comment:
This is a generated code, so I don't think we should change this here.

Reply:
Do you want me to drop this file?

Reply:
I can't figure out where that's coming from :-(

Reply:
Ok, looks like it's https://github.com/sbt/contraband/blob/7f0e6c4765b6dc1ca3534f051d729d452890d5e0/library/src/test/scala/GraphQLCodecCodeGenSpec.scala#L16